docs/sources/getting-started/add-preconfigured-dashboard.md

```diff
@@ -1,33 +1,32 @@
 +++
 aliases = ["/docs/grafana/latest/getting-started/getting-started/", "/docs/grafana/latest/guides/getting_started/", "/docs/grafana/latest/guides/gettingstarted/"]
-description = "Guide for getting started with Grafana"
-keywords = ["grafana", "intro", "guide", "started"]
-title = "With Grafana"
+description = "Learn how to get started with Grafana by adding a preconfigured dashboard"
+title = "Add a preconfigured dashboard"
 weight = 200
 +++
 
-# Getting started with Grafana
+# Add a preconfigured dashboard
 
 This topic helps you get started with Grafana and build your first dashboard. To learn more about Grafana, refer to [What is Grafana?]({{< relref "_index.md" >}}).
 
 > **Note:** Grafana also offers a [free account with Grafana Cloud](https://grafana.com/signup/cloud/connect-account?pg=gsdocs) to help getting started even easier and faster. You can install Grafana to self-host or get a free Grafana Cloud account.
 
-## Step 1: Install Grafana
+## Install Grafana
```

Comment:
I think we should talk about creating a template - I read your comments in mine - and think we could come to a consensus for consistency so these docs have a similar look and feel.